client/src/containers/CreateCardFormContainer/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import CreateCardForm from 'src/components/CreateCardForm'
+
+// import styles from './styles.css';
+
+class CreateCardFormContainer extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      description: '',
+    }
+
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+  render = () => (
+    <CreateCardForm
+      description={this.description}
+      handleChange={this.handleChange}
+      handleSubmit={this.handleSubmit}
```

Comment:
@DDKatch community naming for such events are with preposition `on...`

So:

* `onChange ` instead of `handleChange`
* `onSubmit` instead of `handleSubmit`

Reply:
@AleksSenkou should we rename this props with onChange or setDescription | onSubmit or createCard ?